semgrep-core/dune

```diff
@@ -13,3 +13,125 @@
     (flags (:standard  -w -52-6)))
   (release
     (flags (:standard -O3))))
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+; Don't build executables for the submodules that we only use as libraries.
+; - Saves a tiny bit of build time (about 3-4%).
+; - Keeps the bin/ folder clean.
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+(subdir
+   src/pfff
+   (data_only_dirs cli)
```

Comment:
Actually I use the binary from pfff inside semgrep, because it has more options to dump ASTs, like
-dump_ml, -dump_java, -dump_php, which dumps the original ASTs of the language, not the generic AST one.

But I guess I can clone pfff in its own place and use the binary built from there.

Reply:
I wasn't sure about pfff. I was mostly annoyed when calling `ocaml-tree-sitter` because the `semgrep-core/bin` was in my PATH so I could call `semgrep-core` without installing. When calling `ocaml-tree-sitter`, it would find `semgrep-core/bin/ocaml-tree-sitter` which is usually not the version I want.